_data/projects/PyTorch

```diff
@@ -0,0 +1,14 @@
+name: PyTorch
+desc: PyTorch is an open source machine learning library, used for applications such as computer vision and NLP
+site: https://pytorch.org/
+tags:
+- python
+- c++
+- cuda
+- machine learning
+upforgrabs:
+  name: help needed
+  link:https://github.com/pytorch/pytorch
```

Comment:
There is no label by the name of `help needed`, please check here https://github.com/pytorch/pytorch/labels
Please add a label which is available or create one. You can use 'good first issue' or 'help wanted' which are mentioned there.